### DISCUSSION TITLE
Align all display rotation orientation with display FPC

### DISCUSSION THREAD

**Discussion by moononournation** [moononournation/Arduino_GFX#665] [General]
This is an never closing discussion migrated from original issue #46.

I would like to align all display rotation orientation with display FPC. But I am still struggling the display FPC should be at 6 o'clock or 12 o'clock.
At the beginning I believe the FPC should be at 12 o'clock. However, I found many breakout board silk print text told me the FPC should be at 6 o'clock.
Welcome to leave your comment here.

**Comment by nicknugget** [comment]
I checked all my OLED and TFT modules I have laying around and I would go for 6 o'clock. As most are oriented that way and have the FPC at the bottom. Except one 0,96" 80x160 TFT which has the FPC at 9 o'clock.
